GOVERNANCE.md

```diff
@@ -0,0 +1,107 @@
+# Process for becoming a maintainer
+
+Depending on the status of your current organization, the process for becoming a maintainer may differ as described below:
+
+## Your organization is not yet a maintainer
+
+1. Express interest to the senior maintainers that your organization is interested in becoming a
+  maintainer. Becoming a maintainer generally means that you are going to be spending substantial
+  time (>25%) on Envoy for the foreseeable future. You are expected to have domain expertise and be extremely

```

Comment:
```suggestion
  time (>25%) on Envoy for a foreseeable future. You are expected to have domain expertise and be extremely
```

Reply:
"For the forseeable future" is a turn of phrase https://www.merriam-webster.com/dictionary/in/for%20the%20foreseeable%20future